heroku/cli
# refactor: migrate pg:info to cli/core

[Work item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bDEMdYAO/view)

## Notes
- Please take a close look at the logic, especially the displayDB function. I had to do a bit of refactoring on that one because of Typescript requirements.

## Testing
- Check out this branch and run `yarn` and `yarn build`
- Find or set up an app with a postgres database
- Run `./bin/run pg:info --help` and verify that you get the expected result
- Run `./bin/run pg:info -a APP_NAME DATABASE_NAME`. Verify that you get the expected result.
